mysql-test/main/mysqladmin.test

```diff
@@ -1,5 +1,7 @@
 # Embedded server doesn't support external clients
 --source include/not_embedded.inc
+# MDEV-37169 - msan unknown failure
```

Comment:
P.S. A nit: I think the capitalization is `MSan`? It doesn’t really matter; this reason shouldn’t stay for the long term.

Reply:
.oO(nothing is more permanent than the temporary)